packages/angular/projects/ui-angular/src/lib/primitives/button/button.component.ts

```diff
@@ -28,6 +29,10 @@ export class ButtonComponent implements OnInit {
     if (this.fullWidth) {
       className += ` amplify-button--fullwidth`;
     }
+    if (this.disabled) {
```

Comment:
I could be wrong but why not use `classnames`(https://www.npmjs.com/package/classnames) to get rid of those `if` conditions?

Reply:
Ooh, we've never seen that package before tbh. What do you think @SantoshLatpate? 

Doesn't have to be this PR, but looks like it can simplify our code a little.

Reply:
I will look into classnames and if possible try to get rid of those if conditions